include/localPRG.h

```diff
@@ -39,6 +39,8 @@ class LocalPRG {
     //VCF vcf;
     std::vector<uint32_t> num_hits;
 
+    static bool path_memoization_in_nodes_along_path_method; //flag to turn on or off path memoization in nodes_along_path method
```

Comment:
Given this is a `bool` maybe the name should be `do_path_memoization_in_nodes_along_path_method` (would also mean you can remove the comment)

Reply:
Done in new commit